moped-editor/src/views/projects/projectView/ProjectContractsTable.js

```diff
@@ -23,6 +23,8 @@ import MaterialTable, {
 import typography from "../../../theme/typography";
 
 import { PAGING_DEFAULT_COUNT } from "../../../constants/tables";
+import { currencyFormatter } from "../../../utils/numberFormatters";
+import FundingAmountIntegerField from "./FundingAmountIntegerField";
```

Comment:
Maybe this could be renamed to DollarIntegerField (or something to that effect) now that it is being used outside of funding? Either way, just glad to see this component get reused 🚀